OpenShift.md

```diff
@@ -0,0 +1,75 @@
+# Setup on OpenShift
+
+This example will use [minishift](https://docs.openshift.org/latest/minishift/getting-started/index.html).
+First step is to start this environment:
+
+```
+minishift start --openshift-version=v3.6.0-alpha.2
+```
+
+You can use a more [recent version](https://github.com/openshift/origin/releases) if available.
+
+When fully started, then launch the dashboard using the link displayed following `The server is accessible via web console at:
+`. Navigate to the `New Project` overview page.
+
+NOTE: After using `oc create -f ...` to deploy something to Kubernetes, use the console to check that it
```

Comment:
Yes, in the openshift console, login as developer/developer - it is only the command line that needs to login as admin. I'll update the instructions.